cmd/cronjob-checker/main.go

```diff
@@ -81,12 +90,11 @@ func main() {
 		lastRunTimeV1 := cronGet.Status.LastScheduleTime
 		lastRunTime := lastRunTimeV1.Time
 		shouldOfRun := findLastCronRunTime(schedule)
-		timeMinus := shouldOfRun.Add(time.Duration(-30) * time.Second)
-		timePlus := shouldOfRun.Add(time.Duration(30) * time.Second)
+		tM, tP := scheduleWindow(shouldOfRun, time.Minute*5)
```

Comment:
`tM` and `tP` are a little ambiguous here.  Could we rename this to something like `earliestRunTime` and `latestRunTime`

Reply:
done